client/my-sites/plugins/plugin-site-disabled-manage/index.jsx

```diff
@@ -3,16 +3,18 @@
  */
 import React from 'react';
 import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import analytics from 'lib/analytics';
 import Button from 'components/button';
 import DisconnectJetpackButton from 'my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button';
+import { getJetpackSiteRemoteManagementUrl } from 'state/sites/selectors';
 
-const PluginSiteDisabledManage = ( { isNetwork, site, plugin, translate } ) => {
-	const url = site.getRemoteManagementURL() + '&section=plugins';
+const PluginSiteDisabledManage = ( { isNetwork, site, remoteManagementUrl, plugin, translate } ) => {
```

Comment:
When we have more than a couple of props, it can be more readable if we keep them on separate lines. Would you like to do it here?